ru/turing/courses/lesson2/Yemelyanov/Homework2.java

```diff
@@ -0,0 +1,70 @@
+package ru.turing.courses.lesson2.Yemelyanov;
+
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class Homework2 {
+    public static void main(String[] args) {
+        System.out.println("Введите дату рождения персонажа: ");
+        Scanner input = new Scanner(System.in);
+
+        String inputDate = input.nextLine();
+        System.out.println("Введите адрес в формате: (страна: -, город: -, улица: -, дом: -, квартира: -.)");
+
+        String adressByFormat = input.nextLine(); //Ввод адреса по формату
+        DateTimeFormatter validDateFormat = DateTimeFormatter.ofPattern("dd.MM.yyyy"); //Конвертация введенной даты по формату
+
+        LocalDate birthDay = LocalDate.parse(inputDate, validDateFormat); //День рожжения по формату
+        Person person = new Person(birthDay, adressByFormat);
+
+        UserProcessor personInfo = new UserProcessor();
+
+        String[] adressParts = new String[5]; //Инициализацяи массива строк для заполнения полей адреса
+        adressParts = personInfo.convertAdress(person); //Парсинг адреса
```

Comment:
Это можно сделать в одну строчку
```suggestion
        String[] adressParts = personInfo.convertAdress(person); //Парсинг адреса
```